maidsafe/safe_browser
# fix(tabs): updates tab with URL when navigating via anchor link

Resolves #529 

QA:
- execute via `yarn prod-dev`
- Navigate to `safe://safenetworkprimer`
- Click on `http` anchor link at bottom of page
- Correct URL should appear in notification
